packages/react/src/components/Accordion/Accordion.module.css

```diff
@@ -49,6 +48,10 @@
   margin: 0;
 }
 
+.border button {
```

Comment:
This will have an unintended effect for  all buttons inside the `Accordion`. Suggestion changing it to `.border .header > button`, will probably have to be placed after `.header` declaration also.

Reply:
I made it super safe and added a `className` to `button` 😄